### ISSUE TITLE
Misleading description

### ISSUE THREAD

**Issue by eimann** [SamyPesse/tv.js#4]
The description states "Apple TV for Torrent Streaming in JS (Node/Chrome)" yet it is unclear how it does make use of an Apple TV.

**Comment by AaronO** [comment]
Yes, maybe the wording should be improved some people mistakingly read "Torrent Streaming for Apple TV" instead of "Apple TV for Torrent Streaming"

Basically the message that the current description aims to convey is that it's a simple easy to use app/UI to stream torrents and watch them on a Smart TV, and that it brings a form of simplicity similar to Apple's TV.

Let me know how you think we could improve this

**Comment by joaosoares** [comment]
What about "TV-Ready Torrent Streaming JS app"?

**Comment by Andrew565** [comment]
Could also just add "Like" to the beginning. "Like Apple TV... for Torrent Streaming."

**Comment by qq99** [comment]
> What about "TV-Ready Torrent Streaming JS app"?

:+1: for this, perhaps add a description that it aims to provide a UI like that of Netflix or Apple TV
